### PR TITLE
Changing the Control and Dialog for the Rug Plot Dialog

### PR DESCRIPTION
**Pull request by EstherNjeriLiberatta**
@rdstern , I have made the changes you requested according to PR #6766 

- I have changed the label for the control `Variable (Optional)` to` Points (Optional)`.
- I have changed the `Heat Map Options` control label to `Tile Options`.
- I had already renamed the dialog to `Heat Map `, however not appearing on runtime because of translation issue. 

@lloyddewit , I have made the changes by renaming the controls and the dialog, but they can't appear on the run-time because of the translation issue. Kindly resolve it so that they appear on the run-time.

### CODE REVIEW

**Comment by EstherNjeriLiberatta** [comment]
Hello @rdstern @shadrackkibet This fixes the issue of the name of the dialog from `Rug plot` to `Heat map.`

**Comment by shadrackkibet** [comment]
@EstherNjeriLiberatta I really don't understand why you have 124 files changed.

**Comment by EstherNjeriLiberatta** [comment]
@shadrackkibet Changing the name from dlgRugPlot to dlgHeatMapPlot brought all these changes.

File: instat/dlgMyFirstDialog.vb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+    End Sub
+
+    Private Sub ucrBase_Load(sender As Object, e As EventArgs) Handles ucrBase.Load
+
+    End Sub
+
+    Private Sub dlgMyFirstDialog_Load(sender As Object, e As EventArgs) Handles MyBase.Load
+
+    End Sub
```
Comment (shadrackkibet) [inline instat/dlgMyFirstDialog.vb:12]: Including this new dialog?

File: instat/Form1.vb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿Public Class Form1
+
+End Class
```
Comment (shadrackkibet) [inline instat/Form1.vb:3]: This as well?

**Review by shadrackkibet — APPROVED** [review]

File: instat/dlgMyFirstDialog.vb (hunk 1 above)
Comment (EstherNjeriLiberatta) [reply instat/dlgMyFirstDialog.vb:12]: Ooh no, sorry. I was trying to delete that. It was the first dialog I created.

File: instat/Form1.vb (hunk 2 above)
Comment (EstherNjeriLiberatta) [reply instat/Form1.vb:3]: Ooh God, how can I undo that?

Comment (EstherNjeriLiberatta) [reply instat/Form1.vb:3]: I was deleting it as well

**Review by rdstern — CHANGES_REQUESTED** [review]
@EstherNjeriLiberatta that's looking good now.  I don't see the need to repeat the word Points twice.  Perhaps get rid of the group box, because there is now just the single control inside it.  Then I have made some more major suggestions for improvements in #6665, but would like to get the dialogue merged first in its current form.

**Comment by EstherNjeriLiberatta** [comment]
@rdstern Okay, I will look at it.

**Comment by EstherNjeriLiberatta** [comment]
@rdstern @shadrackkibet this is ready for review.

**Review by rdstern — APPROVED** [review]
@EstherNjeriLiberatta this looks fine now.  I am approving in case @shadrackkibet would like to merge as it is.  But I realise that it doesn't yet work with the option to include multiple y-variables.  It can be made to work and here is the code:
'''heatmap <- data_book$get_data_frame(data_name="heatmap", stack_data=TRUE, measure.vars=c("no.input","fert.1","fert.2","organic.fert","leg.2","leg.1","leg.4","leg.5","leg.3","leg.7","leg.6"))
last_graph <- ggplot2::ggplot(data=heatmap, mapping=ggplot2::aes(x=sitefm., fill=value,y=variable)) + ggplot2::geom_tile() + theme_grey()+coord_flip()
data_book$add_graph(graph_name="last_graph", graph=last_graph, data_name="heatmap")
data_book$get_graphs(data_name="heatmap", graph_name="last_graph")
rm(list=c("last_graph", "heatmap"))'''

I tried with the same date I have now included.  I read in the data in the unstacked shape and can show you how if need be.. You need to do the following:
a) Allow the x to be completed, that's the other factor.
b) Put value into the fill control.  (You will see that is done  with variable in some other dialogues.
c) Generate the code where the y = variable, fill = value. as in the code above.
d) Also have a checkbox on the left with coord_flip.  Call it Swap x and y.  Generate coord_flip() when it is checked.  (You will see that on some other dialogues.

If this is too difficult for now, then we could merge as you have it.  In that case, copy this suggestion into a new issue, or combine it with the other suggestions I have made in #6665

**Comment by EstherNjeriLiberatta** [comment]
@shadrackkibet Please allow me to I finish with these changes.

**Comment by EstherNjeriLiberatta** [comment]
Hello @shadrackkibet I realised I need more time to work on these changes, please could you merge this. I'll add it as a new issue.

**Comment by shadrackkibet** [comment]
@EstherNjeriLiberatta thanks for the notification. I am still not confident with merging this PR in the current state. Let us get a second opinion from @lloyddewit. Points to understand;

- Why there are so many file changes.
- There are two files that should be removed - `dlgMyFirstDialog` and `Form1`. 
- I see changes from `dlgReorderLevels` dialog. Did you intend to make these changes?

**Comment by EstherNjeriLiberatta** [comment]
@shadrackkibet No, I had not intended to make these changes.

**Pull request closed without merging**